### PR TITLE
ERT, Lighting map fixes

### PR DESCRIPTION
**Pull request by Probe1**
- Moves ERT to the Ready Room instead of the deathsquad room.  Changes ERT landmark from "Commando" to "ERT" and adds "ERT" landmarks to the ready room on Box and Metaclub.  Taxi and Deff were not changed because TAXI AND DEFF DONT HAVE ERT ROOMS which is a PR for another day and one Deity has said he wants to work on personally.
- Adds about 35 lights across all maps and z levels to illuminate areas that were unreasonably dark.  The locations are listed below in the comments.
- Removes an accidental lever someone placed inside of a wall right above mining EVA on taxi.
- Adds a few wires leading to an apc on Deff that some wonderful frenchman forgot.

Please review my changes.  I looked at the code, took a guess at how it worked then changed it, compiled, and tested it.  It worked but hey maybe I've done something horrible and I don't realize it.

Partially addresses https://github.com/d3athrow/vgstation13/issues/5940
Closes https://github.com/d3athrow/vgstation13/issues/5938
Addresses https://github.com/d3athrow/vgstation13/issues/5915

### CODE REVIEW

File: maps/metaclub.dmm [hunk 1]
```diff
@@ -6266,13 +6266,13 @@
...
-"cqC" = (/obj/machinery/atmospherics/pipe/simple/supply/hidden,/obj/structure/disposalpipe/segment,/obj/machinery/door/firedoor/border_only,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/Doorresearch.dmi'; id_tag = ""; name = "Research Division"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/effect/decal/warning_stripes{icon_state = "unloading"},/turf/simulated/floor{dir = 4; icon_state = "floor"},/area/science/hallway{name = "Research Division"})
-"cqD" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,/obj/machinery/door/firedoor/border_only,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/Doorresearch.dmi'; id_tag = ""; name = "Research Division"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/effect/decal/warning_stripes{icon_state = "unloading"},/turf/simulated/floor{dir = 4; icon_state = "floor"},/area/science/hallway{name = "Research Division"})
```
Comment (Probe1) [inline maps/metaclub.dmm]: I have no idea why these changes were made but PJB says it's alright.  Please review this and make sure smartgit didn't fuck my shit up.

File: maps/metaclub.dmm [hunk 2]
```diff
@@ -6266,13 +6266,13 @@
...
-"cqI" = (/obj/machinery/atmospherics/pipe/simple/supply/hidden,/obj/structure/disposalpipe/segment,/obj/machinery/door/firedoor/border_only,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/Doorresearch.dmi'; id_tag = "ResearchFoyer"; name = "Research Division"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/effect/decal/warning_stripes{icon_state = "unloading"},/turf/simulated/floor{dir = 4; icon_state = "floor"},/area/science/hallway{name = "Research Division"})
+"cqI" = (/obj/machinery/door/firedoor/border_only,/obj/structure/disposalpipe/segment,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/doorresearch.dmi'; id_tag = ""; name = "Research Break Room"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,/turf/simulated/floor{tag = "icon-cafeteria (NORTHEAST)"; icon_state = "cafeteria"; dir = 5},/area/science/hallway{name = "Research Division"})
```
Comment (Probe1) [inline maps/metaclub.dmm]: Ditto

File: maps/metaclub.dmm [hunk 3]
```diff
@@ -5998,8 +5998,8 @@
...
+"clu" = (/obj/machinery/atmospherics/pipe/simple/supply/hidden,/obj/structure/disposalpipe/segment,/obj/machinery/door/firedoor/border_only,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/doorresearch.dmi'; id_tag = "ResearchFoyer"; name = "Research Division"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/effect/decal/warning_stripes{icon_state = "unloading"},/turf/simulated/floor{dir = 4; icon_state = "floor"},/area/science/hallway{name = "Research Division"})
+"clv" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,/obj/machinery/door/firedoor/border_only,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/doorresearch.dmi'; id_tag = ""; name = "Research Division"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/effect/decal/warning_stripes{icon_state = "unloading"},/turf/simulated/floor{dir = 4; icon_state = "floor"},/area/science/hallway{name = "Research Division"})
```
Comment (Probe1) [inline maps/metaclub.dmm]: Why did smartgit change these files?  PJB says its fine but I dont' understand it and that makes me nervous.

File: maps/metaclub.dmm (hunk 1 above)
Comment (PJB3005) [inline maps/metaclub.dmm]: Shouldn't be smartgit, sounds like the map merger dun fucked up.

File: maps/metaclub.dmm (hunk 2 above)
Comment (PJB3005) [inline maps/metaclub.dmm]: ditto

File: maps/metaclub.dmm (hunk 3 above)
Comment (PJB3005) [inline maps/metaclub.dmm]: ditto

**Comment by Rei1226** [comment]
Wait, are these also fixes to the dark spots on the maps? Says lighting changes, but seems to be mostly ERT stuff or am I just misinterpreting something

**Comment by Probe1** [comment]
No, that's coming.  I added 6 lights to the station before deciding this is fucking stupid and we should push up the lighting value closer to what it was instead of adding an assload of lights.

There are still a lot of places (like Deff arrivals) that need lights but to be honest I'd rather not add a lot of lights when we can tweak a value and wash our hands.

**Comment by PJB3005** [comment]
If they need changing, then change them, it's only a couple values in the code.

**Comment by Probe1** [comment]
I changed them but saw no effective change so I'm returning to Plan A:  Add lights.

**Comment by Probe1** [comment]
Box: 
Surgery Overflow back door
Research Div East Access next to RnD
Mech Bay next to the cyborg chargers
Chapel next to the mass driver
Theatre above the disposal bin
Research Outpost temporary storage.  It's left of the conveyors.
Mining Outpost: Gas Miner

**Comment by Probe1** [comment]
Deff:
Arrivals table fine dining cigarette lighter area.
Mechanics back wall next to the maint door.
Morgue SW corner
Surgery Observation
Research Shuttle (I don't want to know how lights work on it and will likely have to revisit this to varedit it)
Two in toxin storage
Replaces one in the toxin mixing room hall to the observation room with a tube light
Three in the library near shelves and over the photocopier
One in cargo crate storage
2 in atmos, 2 in atmos storage

**Comment by Probe1** [comment]
Taxi:
One in AI Upload
Replaces MoMMI bulb with tube
One in the NE corner of morgue
One in Engineering-Medical the power room on Medstation
SW corner of engineering storage on Engineering Station
Replaces Librarian bulb with a tube
Gets rid of 3 bulbs on the asteroid and replaces them with hanging lanterns.  Come on guys.



Unrelated to lights - gets rid of a conveyor someone accidentally put inside of a wall behind mining EVA.  Wasn't linked to any conveyor nearby.  I assume someone misclicked.

**Comment by Probe1** [comment]
Meta:
Bathrooms, bulb north side
Engineering AME Hallway, bulb north center
Mining Outpost Oxygen Miner, bulb (same deal as Box)
Research Outpost Exterior Access, two tubes
North Mining Outpost, one tube

Congratulations on being well lit Metaclub!
